test/unit/spec/EnrollChoices_spec.js

```diff
@@ -377,23 +377,23 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
             test.form.submit();
             return tick();
           })
-          .then(function () {
-            expect($.ajax.calls.count()).toBe(1);
-            Expect.isJsonPost($.ajax.calls.argsFor(0), {
-              url: 'https://foo.com/api/v1/authn/skip',
-              data: {
-                stateToken: 'testStateToken'
-              }
+            .then(function () {
+              expect($.ajax.calls.count()).toBe(1);
+              Expect.isJsonPost($.ajax.calls.argsFor(0), {
+                url: 'https://foo.com/api/v1/authn/skip',
+                data: {
+                  stateToken: 'testStateToken'
+                }
+              });
             });
-          });
         });
         itp('does not show the Finish button if there are no required factors and no ' +
             'optional factors have been enrolled',
-          function () {
-            return setupWithAllOptionalNoneEnrolled().then(function (test) {
-              expect(test.form.submitButton().length).toBe(0);
-            });
+        function () {
+          return setupWithAllOptionalNoneEnrolled().then(function (test) {
+            expect(test.form.submitButton().length).toBe(0);
```

Comment:
nit: I wonder if we can add some linting rule to force the use some syntactic sugar in tests expectations (ex.: `.toHaveLength(0)`).

Reply:
Is also part of `jasmine-jquery`

Reply:
hmm, interesting! will dig into a little bit after `grunt` deprecation.